kernel/aster-nix/src/fs/utils/inode.rs

```diff
@@ -427,6 +427,22 @@ impl dyn Inode {
             offset: from_offset,
         }
     }
+
+    pub fn read_bytes_at(&self, offset: usize, buf: &mut [u8]) -> Result<usize> {
+        self.read_at(offset, &mut VmWriter::from(buf).to_fallible())
```

Comment:
Please do not perform two non-trivial operations on one line. This will add mental burden to the reader.

The following form adds one extra line, but consumes less time to comprehend.

```rust
        let mut writer = VmWriter::from(buf).to_fallible();
        self.read_at(offset, &mut writer)
```

Please modify all locations according to this pattern.

Reply:
Make sense and done